src/Regridder.jl

```diff
@@ -409,14 +409,15 @@ function update_masks!(cs::CoupledSimulation)
     # static mask
     land_s = cs.surface_masks.land
 
-    cs.surface_masks.ice .= min.(ice_d .+ land_s, FT(1)) .- land_s
-    cs.surface_masks.ocean .= (FT(1) .- cs.surface_masks.ice .- land_s)
+    cs.surface_masks.ice .= min.(ice_d, FT(1) .- land_s)
+    cs.surface_masks.ocean .= max.(FT(1) .- (cs.surface_masks.ice .+ land_s), FT(0)) # max needed to avoid Float32 errors
 
     @assert minimum(cs.surface_masks.ice) >= FT(0)
     @assert minimum(cs.surface_masks.land) >= FT(0)
     @assert minimum(cs.surface_masks.ocean) >= FT(0)
```

Comment:
LGTM - maybe include a snippet of the code to show what the problem was initially (generated `-2.98e-8` values for the `cs.surface_masks.ocean` variable while `eps(Float32)` is `1.1920929f-7`, unless this is already tracked elsewhere in an issue). The assertion could be relaxed with `eps(FloatType)` criterion but it looks like we want strict >=0 FT(0) checks here.

Reply:
Linked Issue #271

Reply:
Sounds good, thank you for the review! I already referred to #271 in the buildkite pipeline file, but it's a great idea to comment in the `Regridder.jl` too.